src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -0,0 +1,482 @@
+                                    </Select>
+                                </FormControl>
+                            </Grid>
+                        </TableCell>
+                        <TableCell style={{ width: 130 }}>
+                            <Grid container>
+                                <Grid>
+                                    Action
+                                </Grid>
+                                <Grid>
+                                    <SwapVertIcon onClick={handleActionSort} style={{ color: "grey" }} />
+                                </Grid>
+                            </Grid>
+                            <Grid>
+                                <FormControl
+                                    variant="outlined"
+                                >
+                                    <InputLabel shrink={false} style={{ lineHeight: "5px" }} margin="dense" >{actionType ? "" : "Action Type"}</InputLabel>
+                                    <Select
+                                        id="select"
+                                        labelId="label"
+                                        value={actionType}
+                                        onChange={handleActionSelection}
+                                        style={{ width: 130 }}
+                                        classes={{
+                                            outlined: classes.outlined,
+                                            root: classes.root,
+                                            "MuiInputBase-input": classes.MuiInputBase
+                                        }}
+                                    >
+                                        {actionOptions.map((actionType) =>
+                                            (<MenuItem key={actionType} value={actionType}>
+                                                {actionType}
+                                            </MenuItem>)
+                                        )}
+                                    </Select>
+                                </FormControl>
+                            </Grid>
+                        </TableCell>
+                        <TableCell style={{ width: 115 }}>
+                            <Grid container>
+                                <Grid>
+                                    Object
+                                </Grid>
+                                <Grid>
+                                    <SwapVertIcon onClick={handleObjectSort} style={{ color: "grey" }} />
+                                </Grid>
+                            </Grid>
+                            <Grid>
+                                <FormControl
+                                    variant="outlined"
+                                >
+                                    <InputLabel shrink={false} style={{ lineHeight: "5px" }} margin="dense" >{objectType ? "" : "Object Type"}</InputLabel>
```

Comment:
same thing here - also use the && logic instead of the ? logic `!objecType && "Object Type"`